lib/chef/powershell.rb

```diff
@@ -1,81 +0,0 @@
-#
```

Comment:
Could we either remove this file altogether, if not keep a comment mentioning chef-powershell is now being via gem and point to the powershell-shim repo?